Assets/10_Miscellaneous/03_Settings/CFG_Game Settings.asset

```diff
@@ -55,7 +56,7 @@ MonoBehaviour:
   _autoMap: 0
   _autoMapClearsConsole: 0
   _levels:
-  - {fileID: 11400000, guid: 7b7aac6db6bc67d4180797a86dcde916, type: 2}
+  - {fileID: 0}
```

Comment:
You removed Level 01? This brought up some new issues.

Reply:
I don't get your point everything looks fine on my end. 

![image_2024-08-06_151056591](https://github.com/user-attachments/assets/101c2ace-35e8-4683-b5a4-6c5d907eb2da)

Reply:
These are the changes on the server. Also what I downloaded from the server.

Verify that you uploaded your work properly.

Reply:
Ok, it seems I forgot to commit the changes :P Please check now

Reply:
Sadly not. You just uploaded a font.

Reply:
Now they are here.